pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -41,6 +42,11 @@ const (
 	trustedCABundleConfigMap = "config-trusted-cabundle"
 	clusterInterceptors      = "openshift-pipelines-clusterinterceptors"
 	namespaceVersionLabel    = "openshift-pipelines.tekton.dev/namespace-reconcile-version"
+	createdByKey             = "operator.tekton.dev/created-by"
+	createdByValue           = "RBAC"
+	releaseVersionKey        = "operator.tekton.dev/release-version"
+	targetNamespaceKey       = "operator.tekton.dev/target-namespace"
+	componentName            = "rbacTaskInstallerSet"
```

Comment:
As the `kind` will be `TektonInstallerSet`, can we shorten this name to `rbac` or `rhosp-rbac` or `tekton-rbac`